libs/payments/ui/src/lib/utils/getErrorFtlInfo.ts

```diff
@@ -41,15 +42,15 @@ export function getErrorFtlInfo(
       return {
         buttonFtl: 'next-payment-error-manage-subscription-button',
         buttonLabel: 'Manage my subscription',
-        buttonUrl: `${config.contentServerUrl}/subscriptions${queryParamString}`,
+        buttonUrl: `${config.paymentsNextHostedUrl}/subscriptions/landing${queryParamString}`,
```

Comment:
suggestion: Change buttonUrl to relative path

Since the redirect is to a `payments-next` page, the full URL isn't necessary, and just the relative path should suffice.